db/migrate/20150313054707_create_translations.rb

```diff
@@ -0,0 +1,9 @@
+class CreateTranslations < ActiveRecord::Migration
+  def change
+    create_table :translations do |t|
+      t.datetime :time_assigned
+
```

Comment:
I don't think this will work. You haven't set up the references to the user and video models. Your `belongs_to` relationship won't work.